src/imageTools/probe.js

```diff
@@ -68,6 +68,18 @@ var cornerstoneTools = (function ($, cornerstone, cornerstoneTools) {
         return suv;
     }
     ///////// BEGIN IMAGE RENDERING ///////
+    function pointNearTool(data, coords) {
+        var point = {
+            left: data.handles.end.x,
+            top: data.handles.end.y,
+            width: 0,
+            height: 0
+        };
+
+        var distanceToPoint = cornerstoneMath.rect.distanceToPoint(point, coords);
```

Comment:
Updated the probe's pointNearTool.

BTW man, this library is FREAKING AWESOME. Thanks for open-sourcing it!

It's the kind of lib that empowers you, as a dev, to do things that will 
impress.

This is probably the first commit of many others :)

On 19-May-2015 4:18 PM, Erik Ziegler wrote:
>
> In src/imageTools/probe.js 
> <https://github.com/chafey/cornerstoneTools/pull/41#discussion_r30597263>:
>
> > @@ -68,6 +68,18 @@ var cornerstoneTools = (function ($, cornerstone, cornerstoneTools) {
> >          return suv;
> >      }
> >      ///////// BEGIN IMAGE RENDERING ///////
> > +    function pointNearTool(data, coords) {
> > +        var point = {
> > +            left: data.handles.end.x,
> > +            top: data.handles.end.y,
> > +            width: 0,
> > +            height: 0
> > +        };
> > +
> > +        var distanceToPoint = cornerstoneMath.rect.distanceToPoint(point, coords);
>
> Ah yeah sorry I see what you mean. You should just delete the 
> definition of point and use
>
> var  distanceToPoint=  cornerstoneMath.point.distance(data.handles.end, coords);
>
> Try Sublime Text (http://www.sublimetext.com/) for a good editor. It 
> can help with the spacing issue.
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/chafey/cornerstoneTools/pull/41/files#r30597263>.
>